content/influxdb/v2/migrate-data/migrate-cloud-to-oss.md

```diff
@@ -157,6 +157,7 @@ data = () =>
 rowCount =
     data()
         |> group(columns: ["_start", "_stop"])
+        |> toString()
         |> count()
```

Comment:
Thanks for finding this @paulwer. Updating the change to align with the other versions of this script and use the most performant method. 

```suggestion
        |> count()
        |> group(columns: ["_start", "_stop"])
        |> sum()
```